scapy/config.py

```diff
@@ -757,6 +758,9 @@ class Conf(ConfClass):
     cache_iflist = {}  # type: Dict[str, Tuple[str, List[str], int]]
     #: holds the Scapy IPv4 routing table and provides methods to
     #: manipulate it
+    neighbor = None  # type: 'scapy.layers.l2.Neighbor'
+    # `neighbor` will be filed by scapy.layers.l2
+    #: holds Scapy's neighbor resolution methods
```

Comment:
`#:` makes the comment apply to the lines below it. route should have the "holds the Scapy IPv4...." thing above it